### DISCUSSION TITLE
Ways to underline code in a different color?

### DISCUSSION THREAD

**Discussion by mastevb** [microsoft/vscode-discussions#46] [Extension Development QnA] [answered]
We have yellow and red for warning and error. Can you underline code in a different color, such as blue?

**Comment by alefragnani** [comment, marked as answer]
Hi @mastevb ,

Do you mean that wave underline that appears in editor, which refers to some Problem. Today, it has tree states (Error, Warning and Info), but _oddly_ I don't see any `workbench.colorCustomizations` to customize it. I only see the icons, like `problemsWarningIcon.foreground`.

On the other hand, if that's not the case, you could create your own `Decorator` and customize not only the `color` but `borderStyle`, `borderWidth` and so on. 

Hope this helps
